error_test.go

```diff
@@ -300,24 +310,22 @@ var _ = Describe("Merge", func() {
 		})
 	})
 
-	Context("with a nil argument", func() {
-		const code = "foo"
+	Context("with a nil target", func() {
```

Comment:
The other tests used a similar terminology. However, I think `target` is fine, as it's the `error` that is modified.